ee/tuf/generator/generator.go

```diff
@@ -0,0 +1,153 @@
+package main
```

Comment:
I think this can be (and maybe should be) `package generator`

Reply:
It looks like `package main` is how our existing `ee/ui/assets/generator/generator.go` works, fwiw